packages/one-app-bundler/webpack/loaders/validate-one-app-compatibility-loader.js

```diff
@@ -16,12 +16,12 @@ const loaderUtils = require('loader-utils');
 
 function validateOneAppCompatabilityLoader(content) {
   const { appCompatibility } = loaderUtils.getOptions(this);
-  const match = content.match(/export\s+default\s+(?!from)([\w\d]+)/);
+  const match = content.match(/export\s+default\s+(?!from)([\w\d]+;)/);
```

Comment:
what about
```suggestion
  const match = content.match(/export\s+default\s+(?!from)([\w\d]+;)/).replace(';', '');
```

Reply:
@JAdshead that won't work as it gives a `Cannot read property 'replace' of null"`